README.md

```diff
@@ -1,3 +1,5 @@
+[![Puppet Forge](http://img.shields.io/puppetforge/v/katello/gutterball.svg)](https://forge.puppetlabs.com/katello/gutterball)
```

Comment:
All of Foreman's are HTTP too, https://raw.githubusercontent.com/theforeman/puppet-foreman/master/README.md